src/main/java/com/o19s/es/ltr/ranker/normalizer/Normalizers.java

```diff
@@ -0,0 +1,55 @@
+package com.o19s.es.ltr.ranker.normalizer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class that manages Normalizer implementations
+ */
+public class Normalizers {
+    private static final Map<String, Normalizer> NORMALIZERS = new HashMap<>();
+    public static final String DEFAULT_NORMALIZER_NAME = "noop";
+
+    static {
+        register("noop", new NoopNormalizer());
```

Comment:
nit: I think it'd be more "future proof" if we made NORMALIZERS a Collections.immutableMap(), basically only allow static initialization here. I worry of having a mutable Map here.
If we wanted to make Normalizers easier to register I would go the path of a Builder in the LtrPluginConstructor similar to what we do with LtrRankerParserFactory but this seems overkill for this particular patch.

Reply:
Sure, easy registration does seem unnecessary at this point.